osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/ReverseArrowPiece.cs

```diff
@@ -37,7 +42,10 @@ public ReverseArrowPiece()
             };
         }
 
-        protected override void OnNewBeat(int beatIndex, TimingControlPoint timingPoint, EffectControlPoint effectPoint, TrackAmplitudes amplitudes) =>
-            Child.ScaleTo(1.3f).ScaleTo(1f, timingPoint.BeatLength, Easing.Out);
+        protected override void OnNewBeat(int beatIndex, TimingControlPoint timingPoint, EffectControlPoint effectPoint, TrackAmplitudes amplitudes)
+        {
+            if (!drawableRepeat.IsHit)
```

Comment:
Should it still bounce if it's missed?/Does that even matter?

 Maybe this should be `Judged` instead if not?

Reply:
unless it's doing a different animation on miss, it should definitely continue bouncing